values.yaml

```diff
@@ -3,7 +3,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 spark:
-  enabled: true
+  enabled: false
```

Comment:
@AyadiAmen you have disabled all the services, please enable them and disable the elk-stack.